COMETwebapp/Components/SubscriptionDashboard/SubscribedTable.razor

```diff
@@ -92,20 +93,47 @@ Copyright (c) 2022 RHEA System S.A.
     /// </summary>
     public ObservableCollection<ParameterSubscriptionViewModel> ViewModelSubscribedParameters = new ObservableCollection<ParameterSubscriptionViewModel>();
 
+    /// <summary>
+    /// Listeners for the components to update it with ISession
+    /// </summary>
+    private Dictionary<string, IDisposable> Listeners = new Dictionary<string, IDisposable>();
+
     /// <summary>
     /// Initialize component at first render and after session update
     /// </summary>
     protected override void OnInitialized()
     {
-        this.SubscribedParameters = iterationService.GetParameterSubscriptions(ISessionAnchor.OpenIteration, ISessionAnchor.CurrentDomainOfExpertise);
+        this.ViewModelSubscribedParameters.Clear();
```

Comment:
i would wrap these calls in a private `loadData` method (or a similar name like that`)